content/Registration and Authentication in your Django app with dj-rest-auth part1- Bruno Michetti.md

```diff
@@ -0,0 +1,214 @@
+## Registration and Authentication in your Django app with dj-rest-auth (part 1)
+
+A huge amount of existing applications have registration and authentication for users. Maybe every developer in the world has implemented something related to this in their work or while they learned. After the creation of the [Django REST framework](https://www.django-rest-framework.org/), Django developers started to implement more and more app-level REST API endpoints.  
+As a result, an open source package called [dj-rest-auth](https://dj-rest-auth.readthedocs.io/en/latest/index.html) has been developed to provide a set of REST API endpoints to handle user registration and authentication tasks. In this article divided in two parts, I will talk about this package; and given that it can generate some problems at first, I will give you some tips to avoid them.
+
+### The dj-rest-auth package
+
+If you are a Django developer and you are coding a REST API with authentication, you will find the django-rest-auth package very useful. As said, this package provides a set of REST API endpoints to manage user registration and authentication. After an easy [installation and configuration](https://dj-rest-auth.readthedocs.io/en/latest/installation.html), you will have endpoints for:
+
+- User registration with activation.
+- Login and logout.
+- Retrieve and update the Django User model.
+- Password change.
+- Password reset via e-mail.
+- Social media authentication.
+
+You don't need to do too much work to have those functionalities in your app. Besides, given that dj-rest-auth is an open source package, it has the advantage of being used by lots of programmers that can find errors, purpose improvements, etc. Actually the project is a fork from [django-rest-auth](https://github.com/Tivix/django-rest-auth) that is no longer maintained. So, if you use dj-rest-auth you can be sure that you are using code created and maintained by an open source community.
+
+### Browsable endpoints
+
+Most of the endpoints that dj-rest-auth provides are browsable. This means that if you have added the package to your Django app, you can test those endpoints through the browser. As an example, if you open the browser and put the url of the login endpoint, you will see something like this:
+
+<img src="images/dj-rest-auth-browsable-endpoint.png" alt="Browsable endpoint" width="900"/>
```

Comment:
I think in `.md` the way of import images is like this `![image info](./pictures/image.png)` 🤔

Reply:
This way is ok too. In this way it's easy to define the width of the image

Reply:
I didn't know that, awesome!